.github/workflows/test.yml

```diff
@@ -2,12 +2,136 @@ name: test
 on:
   - push
   - pull_request
+
 jobs:
-  build:
+  install:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'npm'
+      - run: npm install
```

Comment:
I don't understand the point of the install job, since we still have to install the deps in the other jobs.

Reply:
Oh yes you are right it is useless. I thought it would allow to cache node_modules, but I hadn't read the documentation ahah. In fact it's not the case https://github.com/actions/setup-node#caching-global-packages-data

Thanks for the review man, it's very useful 🙏